src/Microsoft.DotNet.Darc/src/DarcLib/VirtualMonoRepo/SourceMappingParser.cs

```diff
@@ -16,20 +16,27 @@ namespace Microsoft.DotNet.DarcLib.VirtualMonoRepo;
 
 public interface ISourceMappingParser
 {
-    Task<IReadOnlyCollection<SourceMapping>> ParseMappings(string path);
+    Task<IReadOnlyCollection<SourceMapping>> ParseMappings();
 }
 
 public class SourceMappingParser : ISourceMappingParser
 {
-    public async Task<IReadOnlyCollection<SourceMapping>> ParseMappings(string vmrPath)
+    private readonly IVmrInfo _vmrConfiguration;
```

Comment:
Maybe name it '_vmrInfo'? So we're consistent between classes (it's `_vmrInfo` in `VmrDependencyTracker`)